### PR TITLE
Update the getting started page to point to QField on the Microsoft Store

### PR DESCRIPTION
**Pull request by nirvn**

### CODE REVIEW

File: documentation/get-started/index.en.md [hunk 1]
```diff
@@ -11,23 +11,25 @@ tx_slug: documentation_get-started_index
 
 === ":material-android: Android"
 
-    QField can be used with any version newer than Android 5, if you are
+    QField can be used with any version newer than Android 6, if you are
     looking to get a new device, we suggest going for at least Android 9.
 
-    [![GooglePlayLink](../assets/images/Get_it_on_Google_play.png)](
+    [![Google Play link](../assets/images/Get_it_on_Google_play.png)](
```
Comment (m-kuhn) [inline documentation/get-started/index.en.md:17]:
```suggestion
    [![Get QField on Google Play](../assets/images/Get_it_on_Google_play.png)](
```

File: documentation/get-started/index.en.md [hunk 2]
```diff
@@ -11,23 +11,25 @@ tx_slug: documentation_get-started_index
...
     https://play.google.com/store/apps/details?id=ch.opengis.qfield)
 
 === ":material-apple: iOS"
 
-    QField can be found on the Appstore.
+    QField can be found on the App Store.
 
-    [![AppstoreLink](../assets/images/app-store.png)](
+    [![App Store link](../assets/images/app-store.png)](
```
Comment (m-kuhn) [inline documentation/get-started/index.en.md:24]:
```suggestion
    [![Get QField on App Store](../assets/images/app-store.png)](
```

File: documentation/get-started/index.en.md [hunk 3]
```diff
@@ -11,23 +11,25 @@ tx_slug: documentation_get-started_index
...
     https://apps.apple.com/app/qfield-for-qgis/id1531726814)
 
 === ":material-microsoft-windows: Windows"
 
-    Download the latest <a href="https://qfield.org/get_latest/?platform=windows" target="_blank">Windows
-    version</a> (x64 64-bit).
+    QField can be found on Microsoft Store.
+
+    [![Microsoft Store link](../assets/images/microsoft-store.png)](
```
Comment (m-kuhn) [inline documentation/get-started/index.en.md:31]:
```suggestion
    [![Get QField on Microsoft Store](../assets/images/microsoft-store.png)](
```

**Comment by nirvn** [comment]
@m-kuhn , suggestions committed.

**Pull request merged**